drivers/bluetooth/hci/h4_ifx_cyw43xxx.c

```diff
@@ -251,19 +251,35 @@ int bt_h4_vnd_setup(const struct device *dev)
 	/* BT settling time after power on */
 	(void)k_msleep(BT_POWER_ON_SETTLING_TIME_MS);
 
+
```

Comment:
Seems like an unnecessary empty line (there should really never be more than one in C code)